libraries _PLUGIN145 ITHO FAN/Itho/IthoCC1101.cpp

```diff
@@ -305,37 +302,37 @@ void IthoCC1101::initReceive()
 	writeRegister(CC1101_AGCCTRL1 ,0x40);
 	writeRegister(CC1101_AGCCTRL0 ,0x91);
 	writeRegister(CC1101_FSCAL3 ,0xA9);
+//	writeRegister(CC1101_FSCAL3 ,0xE9); //DUCO?
 	writeRegister(CC1101_FSCAL2 ,0x2A);
 	writeRegister(CC1101_FSCAL1 ,0x00);
-	writeRegister(CC1101_FSCAL0 ,0x11);
+	writeRegister(CC1101_FSCAL0 ,0x1F);
```

Comment:
Maybe add some comments to why these specific values are chosen

Reply:
The previous plugin was based on a slightly dated Itho library. I have included the latest changes made by supersjimmie on the non plug-in version of the Itho library: https://github.com/supersjimmie/IthoEcoFanRFT/commit/7ae2c9e34e13a628988caa28f975c9929fb9676d

I added a comment on this.